contrib/mkimage-crux.sh

```diff
@@ -14,62 +14,62 @@ die () {
 
 ISO=${1}
 
-ROOTFS=$(mktemp -d ${TMPDIR:-/var/tmp}/rootfs-crux-XXXXXXXXXX)
-CRUX=$(mktemp -d ${TMPDIR:-/var/tmp}/crux-XXXXXXXXXX)
-TMP=$(mktemp -d ${TMPDIR:-/var/tmp}/XXXXXXXXXX)
+ROOTFS=$(mktemp -d "${TMPDIR:-/var/tmp}/rootfs-crux-XXXXXXXXXX")
+CRUX=$(mktemp -d "${TMPDIR:-/var/tmp}/crux-XXXXXXXXXX")
+TMP=$(mktemp -d "${TMPDIR:-/var/tmp}/XXXXXXXXXX")
 
-VERSION=$(basename --suffix=.iso $ISO | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+VERSION=$(basename --suffix=.iso "$ISO" | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
 
 # Mount the ISO
-mount -o ro,loop $ISO $CRUX
+mount -o ro,loop "$ISO" "$CRUX"
 
 # Extract pkgutils
-tar -C $TMP -xf $CRUX/tools/pkgutils#*.pkg.tar.gz
+tar -C "$TMP" -xf "$CRUX"/tools/pkgutils#*.pkg.tar.gz
 
 # Put pkgadd in the $PATH
 export PATH="$TMP/usr/bin:$PATH"
 
 # Install core packages
-mkdir -p $ROOTFS/var/lib/pkg
-touch $ROOTFS/var/lib/pkg/db
+mkdir -p "$ROOTFS/var/lib/pkg"
+touch "$ROOTFS/var/lib/pkg/db"
 for pkg in $CRUX/crux/core/*; do
-    pkgadd -r $ROOTFS $pkg
+    pkgadd -r "$ROOTFS" "$pkg"
 done
 
 # Remove agetty and inittab config
-if (grep agetty ${ROOTFS}/etc/inittab 2>&1 > /dev/null); then
+if (grep agetty "${ROOTFS}/etc/inittab" > /dev/null 2>&1); then
```

Comment:
agree, assuming file exist `grep -q, grep --quiet` is okay,
in case of file not exist, or accidentally removal, maybe keep the original line is better option,

it is original design to be: **no output, even if there's an error**
so there's no issue with original `2>&1 >/dev/null`